projects/dex-ui/src/utils/wagmi/Client.tsx

```diff
@@ -81,13 +81,13 @@ export const client: any = createClient({
         shimDisconnect: true
       }
     }),
-    // new WalletConnectConnector({
-    //   chains,
-    //   options: {
-    //     projectId: "TODO",
-    //     showQrModal: true
-    //   }
-    // }),
+    new WalletConnectConnector({
+      chains,
+      options: {
+        projectId: import.meta.env.VITE_WALLET_CONNECT_PROJECT_ID,
```

Comment:
We should check that this env file exists and either throw an error (breaking the whole app), or conditionally not load the connector if the id isn't set.